gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/ExplodeBundle.java

```diff
@@ -65,11 +65,11 @@ void explodeBundle(String folderPath, FilterConfiguration filterConfiguration, F
         Bundle filteredBundle = bundleFilter.filter(folderPath, filterConfiguration, bundle);
         //Link, simplify and process entities
         final Collection<EntitiesLinker> entityLinkers = entityLinkerRegistry.getEntityLinkers();
-        entityLinkers.parallelStream().forEach(e -> e.link(filteredBundle, bundle, explodeDirectory));
+        entityLinkers.forEach(e -> e.link(filteredBundle, bundle, explodeDirectory));
```

Comment:
Any reason for this change?!

Reply:
@graju256 I want some EntityLinkers to be executed in order as few fields set by 1 EntityLinker is used to later EntityLinker. I have defined this order in `EntityLinkerRegistry`